android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -281,6 +282,22 @@ public void run() {
         });
     }
 
+    public void refreshTitle_fork(){
```

Comment:
Not sure if we want to include `_fork` in these names. That might create a lot more work if we plan to PR this to the `react-native-video` team. I think the comments are a good way to mark it though.

Reply:
Added this to the ReadMe under tech debt